model/schema.sql

```diff
@@ -977,6 +977,72 @@ CREATE SQL SECURITY INVOKER VIEW `org_statistics`
    from organizations org;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `user_app`
+--
+
+DROP TABLE IF EXISTS `user_app`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `user_app` (
+  `uniqueId` varchar(255) CHARACTER SET utf8 NOT NULL,
+  `userId` varchar(255)CHARACTER SET utf8 NOT NULL,,
+  `code` text COLLATE utf8_bin NOT NULL,
+  `state` text COLLATE utf8_bin NOT NULL,
+  `name` text COLLATE utf8_bin default NULL,
+  `description` text COLLATE utf8_bin default NULL,
+  PRIMARY KEY (`uniqueId`),
```

Comment:
I thought we'd make userId an integer, and primary key (userId, uniqueId), because uniqueId is not actually globally unique.